### PR TITLE
docker, docker-ps: add Japanese translation

### PR DESCRIPTION
**Pull request by Kumamoto-Hamachi**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** Unknown

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution. (Just some minor fixes and this PR is GTG)

File: pages.ja/common/docker-ps.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# docker ps
+
+> Dockerコンテナ一覧を表示します。
+> 詳しくはこちら: <https://docs.docker.com/engine/reference/commandline/ps/>
```
Comment (kbdharun) [inline pages.ja/common/docker-ps.md:4]:
```suggestion
> 詳しくはこちら: <https://docs.docker.com/engine/reference/commandline/ps/>.
```

File: pages.ja/common/docker.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# docker
+
+> Dockerコンテナ及びDockerイメージの管理を行います。
+> `docker run`などいくつかのサブコマンドがあります。これらには使用方法についての独自のドキュメントがあります。
+> 詳しくはこちら: <https://docs.docker.com/engine/reference/commandline/cli/>
```
Comment (kbdharun) [inline pages.ja/common/docker.md:5]:
```suggestion
> 詳しくはこちら: <https://docs.docker.com/engine/reference/commandline/cli/>.
```

File: pages.ja/common/docker.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+- イメージからカスタムのコンテナ名でコンテナを起動する:
+
+`docker run --name {{コンテナ名}} {{イメージ}}`
+
+- 既存のコンテナの起動もしくは停止を行う:
+
+`docker {{start|stop}} {{コンテナ名}}`
+
+- dockerレジストリからイメージをプルする
```
Comment (kbdharun) [inline pages.ja/common/docker.md:19]:
```suggestion
- dockerレジストリからイメージをプルする:
```

Comment (Kumamoto-Hamachi) [reply pages.ja/common/docker.md:19]: I overlooked it! I'll respond with additional commits.

File: pages.ja/common/docker-ps.md (hunk 1 above)
Comment (Kumamoto-Hamachi) [reply pages.ja/common/docker-ps.md:4]: Other Japanese translation md files (e.g., 7z.md, chsh.md, etc.) also does not use a trailing dot (.) at the end of the URL. 
Should I fix this?

Comment (kbdharun) [reply pages.ja/common/docker-ps.md:4]: Yeah, probably. Thanks

File: pages.ja/common/docker.md (hunk 3 above)
Comment (Kumamoto-Hamachi) [reply pages.ja/common/docker.md]: Fixed with the following additional commitments!

https://github.com/tldr-pages/tldr/pull/9785/commits/98edc03e12aed1a91e7b2f13ae7a4b248b38269c

File: pages.ja/common/docker-ps.md (hunk 1 above)
Comment (Kumamoto-Hamachi) [reply pages.ja/common/docker-ps.md:4]: Sure. I'll fix it with an additional commit!

Comment (Kumamoto-Hamachi) [reply pages.ja/common/docker-ps.md]: Fixed in https://github.com/tldr-pages/tldr/pull/9785/commits/49628382d32889787e6a8d145afef8cfe5c695e7

File: pages.ja/common/docker.md (hunk 2 above)
Comment (Kumamoto-Hamachi) [reply pages.ja/common/docker.md]: Fixed in https://github.com/tldr-pages/tldr/pull/9785/commits/49628382d32889787e6a8d145afef8cfe5c695e7

**Review by kbdharun — APPROVED** [review]

File: pages.ja/common/docker.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
+# docker
+
+> Dockerコンテナ及びDockerイメージの管理を行います。
+> `docker run`などいくつかのサブコマンドがあります。これらには使用方法についての独自のドキュメントがあります。
```
Comment (blueskyson) [inline pages.ja/common/docker.md:4]: We have a [template](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/subcommand-mention.md#ja) for this.
```suggestion
> `docker run` のようないくつかのサブコマンドには、使用方法についての独自のドキュメントがあります。
```

**Review by blueskyson — APPROVED** [review]

**Pull request merged**